engines/bastion_katello/app/assets/javascripts/bastion_katello/docker-tags/details/views/docker-tag-environments.html

```diff
@@ -0,0 +1,32 @@
+
+    <table bst-table="table"
```

Comment:
You could use the partial table template here if you wanted free search functionality.